### PR TITLE
feat(Tag,TagList): add copiable props

### PR DESCRIPTION
**Pull request by QuiiBz**
## Type

- Feature
#### What is expected?

Add copiable props to `Tag` & `TagList`, similar to `CopyButton`:
- `copiable` is a boolean defining if the tag is copiable or not
- `copyText` is a string to set the tooltip's copy text
- `copiedText` is a string to set the tooltip's copied text

## Relevant logs and/or screenshots

https://github.com/scaleway/scaleway-ui/assets/43268759/c1c7e72c-e3df-49d0-8021-407be0c30674

### CODE REVIEW

File: packages/ui/src/components/Tag/index.tsx [hunk 1]
```diff
@@ -106,5 +126,68 @@ export const Tag = ({
...
+  </>
 )
+
+export const Tag = ({
+  children,
+  isLoading,
+  onClose,
+  icon,
+  copiable = false,
+  copyText = 'Copy',
```
Comment (philibea) [inline packages/ui/src/components/Tag/index.tsx:138]: i don't think we should put default props for translation no ?

Comment (philibea) [reply packages/ui/src/components/Tag/index.tsx:138]: we should make the props required if copiable is true

Comment (QuiiBz) [reply packages/ui/src/components/Tag/index.tsx:138]: We already hardcode these in `CopyButton`

**Review by DorianMaliszewski — APPROVED** [review]

File: packages/ui/src/components/TagList/index.tsx [hunk 2]
```diff
@@ -50,6 +50,9 @@ type TagListProps = {
    * This property define maximum width of each tag. This doesn't apply for tags in tooltip.
    */
   multiline?: boolean
+  copiable?: boolean
+  copyText?: string
+  copiedText?: string
```
Comment (matthprost) [inline packages/ui/src/components/TagList/index.tsx:55]: Better to take those type from Tag directly 

```ts
& Pick<ComponentProps<typeof Tag>, 'copiable' | 'copyText' | 'copiedText'>
```

**Review by matthprost — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/Tag/index.tsx [hunk 3]
```diff
@@ -106,5 +125,68 @@ export const Tag = ({
...
+  copiedText = 'Copied!',
+  disabled,
+  variant = 'neutral',
+  className,
+  'data-testid': dataTestId,
+}: TagProps) => {
+  const [isCopied, setCopied] = useClipboard(
+    typeof children === 'string' ? children : '',
```
Comment (vincentaudebert) [inline packages/ui/src/components/Tag/index.tsx:145]: Should disable copy if it's not string then?

Comment (QuiiBz) [reply packages/ui/src/components/Tag/index.tsx:145]: It's already done line 151

**Review by matthprost — APPROVED** [review]
Nice work 🚀

**Review by PetmePant — APPROVED** [review]

**Pull request merged**